frontend/src/app/questions/page.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Metadata } from "next";
```

Comment:
Can you move this `questions` page under `(pages)`? The `(pages)` directory is supposed to contain all the frontend pages for consistency.

Reply:
sure